astropy/units/format/latex.py

```diff
@@ -82,7 +72,23 @@ def to_string(cls, unit):
                 s = cls.format_exponential_notation(unit.scale) + r"\,"
 
             if len(unit.bases):
-                s += cls._format_bases(unit)
+                if cls.inline:
```

Comment:
So, I guess I liked the old method somewhat better -- I think it is best either to have different classes that do their own thing, or to have a single class with an option.